addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -401,7 +401,13 @@
                 <t t-esc="channel_progress[slide.id].get('quiz_karma_won', 0) if channel_progress[slide.id].get('completed') else channel_progress[slide.id].get('quiz_karma_gain', 0)"/> xp
             </span>
             <span class="badge badge-danger font-weight-normal px-2 py-1 m-1" t-if="not slide.website_published">Unpublished</span>
+            <a t-if="channel.can_upload and not slide.question_ids" t-attf-href="/slides/slide/#{slug(slide)}">
```

Comment:
I think it is better to let the "add quiz" task add the button. If I understand well currently it does nothing and it will be implemented in another task. As features should be as complete as possible when merged let us simply add the preview toggle button here :) .